python/py-astropy/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           python 1.0
 name                py-astropy
-version             3.0.1
+version             3.0.3
 maintainers         {robitaille @astrofrog} {ligo.org:duncan.macleod @duncanmmacleod}
```

Comment:
I'd suggest that you  make this `openmaintainer` unless there is good reason not to.

Reply:
@astrofrog, any objections?

Reply:
I think you can do it, especially as you are now going to be co-maintaining.

Reply:
@pmetzger, see 777685c.